tests/apps/binding_input/app.go

```diff
@@ -18,8 +18,10 @@ import (
 const appPort = 3000
 
 type messageBuffer struct {
-	lock     *sync.RWMutex
-	messages []string
+	lock          *sync.RWMutex
+	messages      []string
+	errorOnce     bool
```

Comment:
is there the reason why we need to get only one error?

Reply:
Added more clarifying comments.